sycl/doc/MultiTileCardWithLevelZero.md

```diff
@@ -19,9 +19,9 @@ by SYCL runtime. Thus in the example above there is CPU (managed by OpenCL backe
 GPUs corresponding to the single physical GPU (managed by either OpenCL or Level-Zero backend).
 There are few ways to filter observable root-devices.
 	
-One is using environment variable SYCL_DEVICE_FILTER described in [EnvironmentVariables.md](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md)
+One is using environment variable ONEAPI_DEVICE_SELECTOR described in [EnvironmentVariables.md](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md)
 ```
-$ SYCL_DEVICE_FILTER=level_zero sycl-ls
+$ ONEAPI_DEVICE_SELECTOR=level_zero sycl-ls

```

Comment:
This is not a valid use of `ONEAPI_DEVICE_SELECTOR`. I believe it should be

```suggestion
$ ONEAPI_DEVICE_SELECTOR=level_zero:* sycl-ls
```

Same applies to the other example below.

Reply:
Also, we should use the official `ext_oneapi_level_zero` spelling.

Reply:
According to [this documentation](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md#oneapi_device_selector), the official spelling is "level_zero".

Reply:
`sycl-ls` uses `ext_oneapi_*` for `level_zero`/`hip`/`cuda`. I would expect that this document is slightly obsolete and needs to be fixed.

Reply:
Currently it shows ext_oneapi_level_zero as below but I will keep sycl-ls until we have official spec change.

[opencl:acc:0] Intel(R) FPGA Emulation Platform for OpenCL(TM), Intel(R) FPGA Emulation Device OpenCL 1.2  [2023.17.11.0.10_160000.prerelease]
[opencl:cpu:1] Intel(R) OpenCL, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz OpenCL 3.0 (Build 0) [2023.17.11.0.10_160000.prerelease]
[opencl:gpu:2] Intel(R) OpenCL Graphics, Intel(R) UHD Graphics 630 OpenCL 3.0 NEO  [23.30.26918.28]
[ext_oneapi_level_zero:gpu:0] Intel(R) Level-Zero, Intel(R) UHD Graphics 630 1.3 [1.3.26918]

Reply:
I checked and the `ONEAPI_DEVICE_SELECTOR` environment variable accepts the string "level_zero" as the name of the backend.  This is also what we show in the documentation.

I think the output of "sycl-ls" is wrong and should be changed.

Reply:
I think the confusion came from the fact that `ONEAPI_DEVICE_SELECTOR` has
```
<backend> ::= { * | level_zero | opencl | cuda | hip | native_cpu }  // case insensitive
```
while 
https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md has
```
enum class backend {
  // ...
  ext_oneapi_level_zero,
  // ...
};
```

SYCL2020 6.3.7. Adding a backend says this:
> The extension should add a new value to the sycl::backend enumeration type using a naming scheme like ext_<vendorstring>_<backendname>. For example, if the Acme vendor adds a backend named "foo", it would add an enumerated value named sycl::backend::ext_acme_foo.

As such, I have two questions:
1) What would happen if we'll have two vendors providing backends with the same name?
2) Why doesn't `native_cpu` follow the naming convention required by the specification (see https://github.com/intel/llvm/blob/7ed894ab0acc8ff09262113fdb08940d22654a30/sycl/include/sycl/backend_types.hpp#L18-L28)?

Reply:
> What would happen if we'll have two vendors providing backends with the same name

Is your question about the `enum class backend` enumeration?  Two vendors would have different "vendor strings" (e.g. ours is "oneapi"), so the enumerators would not collide.

> Why doesn't `native_cpu` follow the naming convention required by the specification 

This is a bug.  @PietroGhg this should be named like `ext_oneapi_native_cpu` instead.  Could you rename this?  I'm not sure if you have existing users that might be depending on this symbol.  If so, you can deprecate the old name and add a new one that is an alias like `ext_oneapi_native_cpu = ext_native_cpu`.

Reply:
Hi @gmlueck, sorry about this, we'll create a PR to fix it soon. We should be able to just rename it, thank you

Reply:
Thanks @PietroGhg!